tuxemon/resources/db/locale/en_US.json

```diff
@@ -576,5 +576,7 @@
     "okaythen": "Okay! Just give me a second and I will heal your Tuxemon!",
     "okaythen2": "Well, it looks like your Tuxemon are full of energy! Here they are!",
 
-    "input_name": "Name?"
+    "input_name": "Name?",
+    "dont_care": "DON'T CARE",
+    "default_names": ["Beep-beep","Toast","Chair","Lizard","Chris","Carter","Kyndi","Wednesday","Tapioca","Pom","Indiana","Kansas"]
```

Comment:
These names are weird

Reply:
I think that's the idea, since they've said that they don't care what their name is 😛 

But a nice Easter egg could be to use the names of Tuxemon contributors! Mircea, Keri, William, etc.